tests/iteration/iteration.test.ts

```diff
@@ -0,0 +1,133 @@
+import createMollieClient, { ApiMode, MollieClient, Payment, PaymentMethod } from '../..';
+import axios from 'axios';
+import nock from 'nock';
+import { setupRecorder } from 'nock-record';
+import { run } from 'ruply';
+import getAccessToken from '../getAccessToken';
+
+const mockNetwork = true;
```

Comment:
Let's add a comment as to what this does

Reply:
Can do!